functests/1_performance/cpu_management.go

```diff
@@ -349,6 +311,73 @@ var _ = Describe("[rfe_id:27363][performance] CPU Management", func() {
 			}
 		})
 	})
+
+	Describe("Verification that IRQ load balance can be disabled per POD", func() {
+		var testpod *corev1.Pod
+
+		BeforeEach(func() {
+			if profile.Spec.GloballyDisableIrqLoadBalancing != nil && *profile.Spec.GloballyDisableIrqLoadBalancing {
+				Skip("IRQ load balance should be enabled (GloballyDisableIrqLoadBalancing=false), skipping test")
+			}
+		})
+
+		AfterEach(func() {
+			deleteTestPod(testpod)
+		})
+
+		It("[test_id:36364] should disable IRQ balance for CPU where POD is running", func() {
+			By("checking default smp affinity is equal to all active CPUs")
+			defaultSmpAffinitySet, err := nodes.GetDefaultSmpAffinitySet(workerRTNode)
+			Expect(err).ToNot(HaveOccurred())
+
+			onlineCPUsSet, err := nodes.GetOnlineCPUsSet(workerRTNode)
+			Expect(err).ToNot(HaveOccurred())
+
+			Expect(defaultSmpAffinitySet).To(Equal(onlineCPUsSet), fmt.Sprintf("Default SMP Affinity %s should be equal to all active CPUs %s", defaultSmpAffinitySet, onlineCPUsSet))
+
+			By("Running pod with annotations that disable specific CPU from IRQ balancer")
+			annotations := map[string]string{
+				"irq-load-balancing.crio.io": "true",
```

Comment:
@cynepco3hahue updated cri-o to use "disable" instead of true, we will need to update these eventually